src/send/README.md

```diff
@@ -0,0 +1,27 @@
+## caliper-event-analysis: Sending the data to the OpenLRS server
+
+This script is mainly written to send caliper events that stored in the Application Database to the OpenLRS server. We are 
+manually extracting the caliper events from the App DB and the script is reading the file containing events one at a time and sends it.
+Sample caliper data file is provided in the repo and this file is Delimited by BREAK. The reason for this is some time the caliper data in the APP DB is stored as pretty-print-json and that 
+leads to adding a new line character when you do a extract. so care must be taken to get raw JSON format.The Caliper extract of each caliper event consist of Envelope info apart from the data. 
+
+### Requirements 
+   1. works with Python 2.7.10 version
```

Comment:
We'll start using Python 3 (most likely 3.6+) more often, so we need to keep compatibility in mind.  Some modules to help with compatibility are [`__future__`](https://docs.python.org/2.7/library/__future__.html) and [`2to3`](https://docs.python.org/2.7/library/2to3.html).  The HOWTO article "[Porting Python 2 Code to Python 3](https://docs.python.org/3/howto/pyporting.html)" is also helpful.

In the little I've done with porting small programs over, the biggest problem is that the `print` statement in 2.7 becomes a `print()` function in 3.  (I believe the 2.7 statement is technically a function, too.)  There's a helper from `__future__` to require parentheses in 2.7, but it's not always necessary.  2.7 will accept arguments for `print` in parentheses, even without the help of `__future__`, which I see you've done here, too.